### DISCUSSION TITLE
Component Library

### DISCUSSION THREAD

**Discussion by thkruz** [thkruz/the-enhanced-scheduling-tool#15] [Ideas]
It only makes sense to use the styling guide Space Force paid some company to make.

https://www.astrouxds.com/

I have used the raw css before but the prebuilt components could be a time saver.

**Comment by tkbdsi** [comment]
Space Force leading the way! Go guys go

**Comment by tkbdsi** [comment]
Pulled all the new styling changes over into my fork and ran them. Very nice. Like the new coloring. Definitely has that space force feel, haha.
